lab_1/main.py

```diff
@@ -12,7 +12,25 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
+    if text and type(text) == str:
+        text = text.lower()
+        text_list = text.split()
+        tokens = []
+        for word in text_list:
+            if word.isalpha():
+                tokens += [word]
+            else:
+                for e in word:
+                    if e in punctuation:
+                        word = word.replace(e, '')
+                        if word.isalpha():
+                            tokens += [word]
+                            break
+                        else:
```

Comment:
Please, shorten this `if`-statement by two lines and explain why and how.

Reply:
I can remove _else_  because it is unnecessary, but that is all I can do, I think. I need _break_ to stop the loop. And I use _tokens.append(word)_ instead.